hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneClientAdapterImpl.java

```diff
@@ -648,4 +663,27 @@ private FileStatusAdapter toFileStatusAdapter(OzoneFileStatus status) {
         status.getPath()
     );
   }
+
+  /**
+   * Generate a FileStatusAdapter for OFS root.
+   * @return FileStatusAdapter for root.
+   */
+  private static FileStatusAdapter rootFileStatusAdapter() {
+    // Note that most fields are mimicked from HDFS FileStatus for root,
+    //  except modification time, permission, owner and group.
+    // TODO: Revisit the return value.
+    return new FileStatusAdapter(
+        0L,
+        null,
+        true,
+        (short)0,
+        0L,
+        0L,
+        0L,
+        (short)0,
+        null,
```

Comment:
use ozone admin

Reply:
But for now we are leaving this field as null since we didn't have an API to retrieve such info from OM yet.